examples/js-pub.c

```diff
@@ -86,9 +86,12 @@ int main(int argc, char **argv)
             // Initialize the configuration structure.
             jsStreamConfig_Init(&cfg);
             cfg.Name = stream;
-            // Set the subject
+            // Set the subject (Two-dimensional array)
             cfg.Subjects = (const char*[1]){subj};
+            // Set the subject count
             cfg.SubjectsLen = 1;
+            // Set the stream memory size
+            cfg.MaxBytes = (1024 * 1024) * 64;
```

Comment:
This example is minimal. There are many ways to restrict the lifetime of messages in a stream, I do not think we should be opinionated in the simple example.

1/5 maybe add a separate example on setting up the stream limits, and remove this line?

Reply:
thank you for the feedback!!

Upon reflection, I agree with the statement "I do not think we should be opinionated in the simple example."
However, if using 'js_MemoryStorage' for storage configuration, it seems necessary to add a comment explaining the message lifespan.

To provide further explanation, in my test environment, although I restricted the memory in the nats-server configuration, not configuring the message lifespan the same way as the current C client example resulted in the nats-server exceeding the configured memory. Additionally, it went beyond the container's total memory limit.

In Korea, there isn't much information on the NATS.io C client, and most of it is in English. So, it took a bit of time to code by referring to the [NATS Docs](https://docs.nats.io/) and [developer documentation](https://nats-io.github.io/nats.c/). I wanted to share the challenging parts I encountered.

Anyway, thank you once again for the feedback.